static/css/_base.scss

```diff
@@ -160,4 +160,12 @@ md-dialog.centered-dialog{
   .active {
     background-color: #E3F2FD;
   }
+}
+span.inset {
+  padding:5px;
+}
+
+.no-surround {
+  margin: 0px !important;
```

Comment:
An alternative to !important is to pay attention to the CSS import order,
or use selectors that are higher precedence (e.g., more selective).

On Thu, Nov 5, 2015 at 11:32 AM, Durim Morina <notifications@github.com>
wrote:

> In static/css/_base.scss
> <https://github.com/crowdresearch/crowdsource-platform/pull/583#discussion_r44057645>
> :
>
> > @@ -160,4 +160,12 @@ md-dialog.centered-dialog{
> >    .active {
> >      background-color: #E3F2FD;
> >    }
> > +}
> > +span.inset {
> > +  padding:5px;
> > +}
> > +
> > +.no-surround {
> > +  margin: 0px !important;
>
> it's highly recommended not to use !important unless you really have to
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crowdresearch/crowdsource-platform/pull/583/files#r44057645>
> .
>